docs/stdlib/array.rst

```diff
@@ -57,15 +57,15 @@ Arrays store expressions of the *same type* in an ordered list.
 Constructing arrays
 ^^^^^^^^^^^^^^^^^^^
 
-An array constructor is an expression that consists of a sequence of
+Array constructors are an expression that allow for a sequence of
 comma-separated expressions *of the same type* enclosed in square brackets.
-It produces an array value:
+This will produce an array of the following:
```

Comment:
Not sure if it helps too much to add clarification on it after constructors are mentioned above this. I can go ahead and add this though!

Reply:
This could just be a misunderstanding on my part of the synopsis block below (and maybe _that's_ where the actual change needs to happen; I don't know). What is the synopsis block below supposed to represent? I thought it was demonstrating the syntax of an array constructor to the user, hence my suggestion. We just need to make sure the text leading up to that colon is introducing what comes after it.

Reply:
I think when it's read altogether it reads out as "this [constructor] will produce an array," and goes in reference to the details above it